app/com/linkedin/drelephant/analysis/Severity.java

```diff
@@ -23,20 +23,23 @@
  * The severities that you can use
  */
 public enum Severity {
+  @EnumValue("5")
```

Comment:
wouldn't this be a breaking change ? You need to update all existing records to new severity values.